defaults/main.yml

```diff
@@ -21,3 +21,25 @@ spark_hive_site_properties: {}
 
 spark_env_extras: {}
 spark_defaults_extras: {}
+
+# Logger levels set to the official defaults
+# Ref: https://github.com/apache/spark/blob/master/conf/log4j.properties.template
+spark_log4j_loggers:
```

Comment:
we could also use a dict instead (the minimally verbose variant):
```
 log4j.logger.org.apache.spark.repl.Main: WARN
 log4j.logger.org.spark_project.jetty: WARN
...
```
or a list of lines
```
- "log4j.logger.org.apache.spark.repl.Main: WARN"
- "log4j.logger.org.spark_project.jetty: WARN"
```

Reply:
I think this is actually OK as is.